src/Control Tasks/RockblockControlTask.cpp

```diff
@@ -922,14 +922,22 @@ void RockblockControlTask::get_valid_signal(rockblock_mode_type good_signal, roc
         Serial.println(signal);
 #endif
         if (signal == '3' || signal == '4' || signal == '5') {
+            bad_signal_checks = 0;
             transition_to(good_signal);
-            signal_checks++;
+            good_signal_checks++;
         } else {
             transition_to(bad_signal);
+            bad_signal_checks++;
         }
 
-        if (signal_checks >= sfr::rockblock::max_signal_check) {
-            Serial.println("SAT INFO: signal checks failed max times");
+        if (good_signal_checks >= sfr::rockblock::max_signal_check) {
+            Serial.println("SAT INFO: good signal checks failed max times");
```

Comment:
you could reset good_signal_checks to 0 here instead of in dispatch_standby for consistency with how you handle bad_signal_checks - or was there a reason for resetting in dispatch standby?

Reply:
Ah ok the reason it was reset in dispatch_standby is in case we hit the case where we are stuck in one mode for ten minutes and go back to standby- in which case should we reset the counter. I will do the same for both counters for consistency.

Reply:
Ah okay got it - makes sense!